Cargo.toml

```diff
@@ -5,8 +5,10 @@ members = [
     "packages/grub",
     "packages/kernel-5.15",
     "packages/kernel-6.1",
+    "packages/kernel-6.12",
     "packages/kmod-5.15-nvidia",
     "packages/kmod-6.1-nvidia",
+    "packages/kmod-6.12-nvidia",
```

Comment:
Please remove the `kmod-6.12-nvidia` package from this PR and let's handle it in a follow up PR.

It's too much change to review repeatedly if the kernel-devel handling in the 6.12 spec file changes as I expect.